Vizzuality/landgriffon
# [LANDGRIF-1169]: Change impact layer resolution according to zoom value

### General description

This PR changes the resolution of the H3 impact layer to increase based on the map zoom value.


When there is a location filter applied:

- In zoom less than 5 it should always render resolution 4
- In zoom between 5 to 7 it should render resolution 5
- In zoom greater than 7 it should render resolution 6

### Testing instructions

Go to analysis/map
Click the zoom in button several times. The map should not be refetched.
Restore the zoom to the minimum value.

Select one or more 'location type' filters. Click the zoom in button 3 times. The map should refetch
(The initial map zoom is 2, so with 3 clicks it goest to zoom 5 and triggers the fetch of impact layer with resolution 5)
Click 3 more times and it should refetch again, now with resolution 6.

Do the inverse process to check if it fetches when the zoom is reduced. 
You can check the payload of the endpoint fetching the layer on network searching for the endpoint '/h3/map/impact'


## Checklist before merging
